Pipfile

```diff
@@ -16,6 +16,8 @@ python-dateutil = "*"
 arrow = "~=1.0.3"
 aiodns = "~=2.0"
 PyJWT = "~=2.1.0"
+InfixParser = "~=3.1.4"
+pybind11 = "*"
```

Comment:
Use exact pinning (`3.1.4`, not `~=3.1.4` or `^3.1.4`) in applications (things that just run standalone, as opposed to libaries).

And especially don't use `*`. Your program will break on the first breaking change of `pybind11`.